sqlmesh/migrations/v0060_mysql_fix_blob_text_type.py

```diff
@@ -0,0 +1,48 @@
+"""Duplicate of v0033, Use LONGTEXT type for blob fields in MySQL.
+
+Seeds table has since been dropped.
+Environments table now has a requirements column.
+"""
+
+from sqlglot import exp
+
+from sqlmesh.utils.migration import blob_text_type
+
+
+def migrate(state_sync, **kwargs):  # type: ignore
+    engine_adapter = state_sync.engine_adapter
+    if engine_adapter.dialect != "mysql":
+        return
+
+    schema = state_sync.schema
+    environments_table = "_environments"
+    snapshots_table = "_snapshots"
+    plan_dags_table = "_plan_dags"
+
+    if schema:
+        environments_table = f"{schema}.{environments_table}"
+        snapshots_table = f"{schema}.{snapshots_table}"
+        plan_dags_table = f"{schema}.{plan_dags_table}"
+
+    targets = [
+        (environments_table, "snapshots"),
+        (environments_table, "previous_finalized_snapshots"),
```

Comment:
I suggest to do the same for `promoted_snapshot_ids`, since it can grow in size quite a bit too

Reply:
Here and in other scripts.